Makefile

```diff
@@ -94,7 +94,9 @@ vet:
 	go vet ./...
 
 .PHONY: build
-build: | $(BINDIR) ## build trust-manager
+build: $(BINDIR)/trust-manager | $(BINDIR) ## build trust-manager for the host system architecture; to cross compile for linux, use "make build-linux-$arch"
```

Comment:
I think this comment does not work for this PR yet, the "build-linux-$arch" target does not exist.